frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -98,20 +99,22 @@ class PublicOrEmbeddedDashboardInner extends Component<PublicOrEmbeddedDashboard
       dashboardId,
     } = this.props;
 
-    initialize();
+    if (!dashboard) {
+      initialize();
```

Comment:
nice, I think this prevents us from initializing twice. I wonder if we should have a test case for this, just to prevent regressions?